Islandora-CLAW/Crayfish-Commons
# Github actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)


# What does this Pull Request do?
Converts to github actions
updates the badges
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)
passed here: 
https://github.com/asulibraries/Crayfish-Commons/actions/runs/794542367
and codecov should be running as well (proof here https://codecov.io/github/asulibraries/Crayfish-Commons/commit/7e122f39d3a929aaaee906f3099780d9a4b47432)

# Interested parties
@Islandora/8-x-committers
